joomla/joomla-cms
# [5.0] schemaorg format json-ld

### Summary of Changes

- use same json flags for breadcrumbs like the schemaorg plugin (see also #41250)
https://github.com/joomla/joomla-cms/blob/a8e363dbd16eac34f79995b08169d9ac9ee23f14/modules/mod_breadcrumbs/tmpl/default.php#L120
https://github.com/joomla/joomla-cms/blob/a8e363dbd16eac34f79995b08169d9ac9ee23f14/plugins/system/schemaorg/src/Extension/Schemaorg.php#L457

- pretty print json like `MetasRenderer`
https://github.com/joomla/joomla-cms/blob/a8e363dbd16eac34f79995b08169d9ac9ee23f14/libraries/src/Document/Renderer/Html/MetasRenderer.php#L71-L72

### Testing Instructions

1. check schemaorg ouput for breadcrumbs
2. enable debug mode

### Actual result BEFORE applying this Pull Request

1.
```JSON
{"@context":"https:\/\/schema.org","@type":"BreadcrumbList","@id":"https:\/\/j50dev.localhost\/#\/schema\/BreadcrumbList\/17"
```

2.
```JSON
{"@context":"https://schema.org","@graph":[{"@type":"Person","@id":"https://j50dev.localhost/#/schema/Person/base","url":"https://j50dev.localhost/"}
```

### Expected result AFTER applying this Pull Request

1.
```JSON
{"@context":"https://schema.org","@type":"BreadcrumbList","@id":"https://j50dev.localhost/#/schema/BreadcrumbList/17"
```

2.

```JSON
{
    "@context": "https://schema.org",
    "@graph": [
        {
            "@type": "Person",
            "@id": "https://j50dev.localhost/#/schema/Person/base",
            "url": "https://j50dev.localhost/"
        }
```

### Link to documentations
Please select:
